### ISSUE TITLE
Training loop

### ISSUE THREAD

**Issue by joh-fischer** [joh-fischer/vision-models#16]
- [x] AdamW
- [x] Learning Rate Scheduler
- [x] Weight decay
- [x] Adapt Readme
- [x] Rerun Experiments
- [x] Update Readme with new results
